app/models/form/sales/subsections/household_characteristics.rb

```diff
@@ -8,7 +8,7 @@ def initialize(id, hsh, section)
 
   def pages
     @pages ||= [
-      Form::Sales::Pages::BuyerInterview.new(nil, nil, self),
+      (Form::Sales::Pages::BuyerInterview.new(nil, nil, self) unless form.start_year_after_2024?),
```

Comment:
There may want to be some refactoring when PrivacyNotice is also moved, but it makes sense to leave it until then.